ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -19,13 +20,14 @@ interface WorkflowLogsViewerState {
 }
 
 export class WorkflowLogsViewer extends React.Component<WorkflowLogsViewerProps, WorkflowLogsViewerState> {
+    public static defaultProps = {follow: true};
```

Comment:
Currently there's not way to actually change this in the UI. Can you add a toggle for it? Telling users to change the URL is not good UX.

Reply:
That would require some design discussions. There maybe other log options that we want to expose. We probably want to experiment a bit and add a configuration UI element (specific type TBD) later in a separate PR.

Reply:
Notwithstanding future changes or design considerations we should add a UX-friendly way to toggle this as this PR would be incomplete without it.

If we find reason to change it in the future, we can do it then.

Reply:
That's true. I'll see what I can do here.

Reply:
Just added a small checkbox to toggle this. This way it is optional and won't affect existing users.